include/z64cutscene.h

```diff
@@ -158,131 +24,488 @@ typedef union CutsceneData {
+    /* 0x003E */ CS_CMD_ACTOR_CUE_8_0,
+    /* 0x003F */ CS_CMD_ACTOR_CUE_3_5,
+    /* 0x0040 */ CS_CMD_ACTOR_CUE_1_6,
+    /* 0x0041 */ CS_CMD_ACTOR_CUE_3_6,
+    /* 0x0042 */ CS_CMD_ACTOR_CUE_3_7,
+    /* 0x0043 */ CS_CMD_ACTOR_CUE_2_4,
+    /* 0x0044 */ CS_CMD_ACTOR_CUE_1_7,
+    /* 0x0045 */ CS_CMD_ACTOR_CUE_2_5,
+    /* 0x0046 */ CS_CMD_ACTOR_CUE_1_8,
+    /* 0x0047 */ CS_CMD_UNIMPLEMENTED_47,
+    /* 0x0048 */ CS_CMD_ACTOR_CUE_2_6,
+    /* 0x0049 */ CS_CMD_UNIMPLEMENTED_49,
+    /* 0x004A */ CS_CMD_ACTOR_CUE_2_7,
+    /* 0x004B */ CS_CMD_ACTOR_CUE_3_8,
+    /* 0x004C */ CS_CMD_ACTOR_CUE_0_7,
+    /* 0x004D */ CS_CMD_ACTOR_CUE_5_2,
+    /* 0x004E */ CS_CMD_ACTOR_CUE_1_9,
+    /* 0x004F */ CS_CMD_ACTOR_CUE_4_5,
+    /* 0x0050 */ CS_CMD_ACTOR_CUE_1_10,
+    /* 0x0051 */ CS_CMD_ACTOR_CUE_2_8,
+    /* 0x0052 */ CS_CMD_ACTOR_CUE_3_9,
+    /* 0x0053 */ CS_CMD_ACTOR_CUE_4_6,
+    /* 0x0054 */ CS_CMD_ACTOR_CUE_5_3,
+    /* 0x0055 */ CS_CMD_ACTOR_CUE_0_8,
+    /* 0x0056 */ CS_CMD_START_SEQ,
+    /* 0x0057 */ CS_CMD_STOP_SEQ,
+    /* 0x0058 */ CS_CMD_ACTOR_CUE_6_4,
+    /* 0x0059 */ CS_CMD_ACTOR_CUE_7_2,
+    /* 0x005A */ CS_CMD_ACTOR_CUE_5_4,
+    /* 0x005D */ CS_CMD_ACTOR_CUE_0_9 = 0x5D,
+    /* 0x005E */ CS_CMD_ACTOR_CUE_1_11,
+    /* 0x0069 */ CS_CMD_ACTOR_CUE_0_10 = 0x69,
+    /* 0x006A */ CS_CMD_ACTOR_CUE_2_9,
+    /* 0x006B */ CS_CMD_ACTOR_CUE_0_11,
+    /* 0x006C */ CS_CMD_ACTOR_CUE_3_10,
+    /* 0x006D */ CS_CMD_UNIMPLEMENTED_6D,
+    /* 0x006E */ CS_CMD_ACTOR_CUE_0_12,
+    /* 0x006F */ CS_CMD_ACTOR_CUE_7_3,
+    /* 0x0070 */ CS_CMD_UNIMPLEMENTED_70,
+    /* 0x0071 */ CS_CMD_UNIMPLEMENTED_71,
+    /* 0x0072 */ CS_CMD_ACTOR_CUE_7_4,
+    /* 0x0073 */ CS_CMD_ACTOR_CUE_6_5,
+    /* 0x0074 */ CS_CMD_ACTOR_CUE_1_12,
+    /* 0x0075 */ CS_CMD_ACTOR_CUE_2_10,
+    /* 0x0076 */ CS_CMD_ACTOR_CUE_1_13,
+    /* 0x0077 */ CS_CMD_ACTOR_CUE_0_13,
+    /* 0x0078 */ CS_CMD_ACTOR_CUE_1_14,
+    /* 0x0079 */ CS_CMD_ACTOR_CUE_2_11,
+    /* 0x007B */ CS_CMD_ACTOR_CUE_0_14 = 0x7B,
+    /* 0x007C */ CS_CMD_FADE_OUT_SEQ,
+    /* 0x007D */ CS_CMD_ACTOR_CUE_1_15,
+    /* 0x007E */ CS_CMD_ACTOR_CUE_2_12,
+    /* 0x007F */ CS_CMD_ACTOR_CUE_3_11,
+    /* 0x0080 */ CS_CMD_ACTOR_CUE_4_7,
+    /* 0x0081 */ CS_CMD_ACTOR_CUE_5_5,
+    /* 0x0082 */ CS_CMD_ACTOR_CUE_6_6,
+    /* 0x0083 */ CS_CMD_ACTOR_CUE_1_16,
+    /* 0x0084 */ CS_CMD_ACTOR_CUE_2_13,
+    /* 0x0085 */ CS_CMD_ACTOR_CUE_3_12,
+    /* 0x0086 */ CS_CMD_ACTOR_CUE_7_5,
+    /* 0x0087 */ CS_CMD_ACTOR_CUE_4_8,
+    /* 0x0088 */ CS_CMD_ACTOR_CUE_5_6,
+    /* 0x0089 */ CS_CMD_ACTOR_CUE_6_7,
+    /* 0x008A */ CS_CMD_ACTOR_CUE_0_15,
+    /* 0x008B */ CS_CMD_ACTOR_CUE_0_16,
+    /* 0x008C */ CS_CMD_TIME,
+    /* 0x008D */ CS_CMD_ACTOR_CUE_1_17,
+    /* 0x008E */ CS_CMD_ACTOR_CUE_7_6,
+    /* 0x008F */ CS_CMD_ACTOR_CUE_9_0,
+    /* 0x0090 */ CS_CMD_ACTOR_CUE_0_17,
+    /* 0x03E8 */ CS_CMD_DESTINATION = 0x03E8,
+    /* 0xFFFF */ CS_CMD_END = 0xFFFF
+} CutsceneCmd;
+
 #define CS_CMD_CONTINUE 0
 #define CS_CMD_STOP -1
```

Comment:
Since these are only for the camera commands I think it may be appropriate to rename them to `CS_CAM_CONTINUE`/`CS_CAM_STOP`? There's also some places that need to use these but don't in other files, search `continueFlag`